src/plugins/intel_gpu/include/intel_gpu/graph/serialization/input_info_serializer.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
nit : 2023

Reply:
I updated the year as you reviewed. Thank you!